ui/app/common/events/events-services.js

```diff
@@ -92,14 +92,18 @@
                 payload: event.payload
               };
 
-              switch (event.payload.buildStatus) {
-                case 'REPO_SETTING_UP':
+              switch (event.payload.buildCoordinationStatus) {
```

Comment:
Leave them all in for now, it would be nice to see what is happening in the background.